src/shortcuts/ShortcutManager.js

```diff
@@ -251,6 +251,8 @@ class ShortcutManager {
 
     getValidChildShortcutsInContext(context, recurse = false) {
         const currentContextId = context.getId();
+        // console.log("current context id: " + currentContextId);

```

Comment:
There's a few commented out console.logs in the file. Do we need to keep those?

Reply:
No, good catch. I removed them and committed